docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Datasets - Reduced data, IRFs, models \n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "`gammapy.datasets` are a crucial part of the gammapy API. `datasets` constitute `DL4` data - binned counts, IRFs, models and the associated likelihoods. `Datasets` from the end product of the `makers` stage, see [makers notebook](makers.ipynb), and are passed on to the `Fit` or estimator classes for modelling and fitting purposes.\n",
+    "\n",
+    "There are three types of `Dataset` available\n",
```

Comment:
Should we add a web link towards the statistics page?

Reply:
Ok